fix typos and add practice

abrilr1604 opened the pull request:
- added file `practice-resolution-maritime-port-activity.ipynb` (please review it)
- corrected typos in files: `chokepoints-monitor.ipynb`, `maritime-port-activity.ipynb`, `port-calls-practice.ipynb`, `port-calls-trends-monitor.ipynb`, `damage-assessment.ipynb`, `visits.ipynb`
- added (Illustrative) to the title of the  notebook `visits.ipynb`
- updated a sentence in the notebook `solar-panel-detection.ipynb` for better clarity
- added cross reference in notebook `damage-assessment.ipynb`
- added image `import_volume_change_chart.png` to the notebook `port-calls-practice.ipynb`

abrilr1604 committed:
fix typos and add practice

msoltadeo commented:
@abrilr1604, I found a way for making cross-references between sections without using the link of the course. I mentioned to you I wanted to do this in one of our meetings some time ago. This is good because if at some point the course is deployed somewhere else and the base url changes, we do not loose the references :muscle:

First, you need to add a label to the section you want to make the reference to. For example, for [this section](https://reimagined-disco-g6q6ny1.pages.github.io/notebooks/mapping-and-monitoring-extent/mapping-and-monitoring-earthquake-intensity.html#overlay-with-admin-level-2) I added:

```
(earthquake-overlay-admin2)=
### Overlay with Admin level 2
Download the data for administrative level 2 for Turkey. [GADM](https://gadm.org/index.html) was used.
```
Second, you quote the reference you had just made. I did this in [this section](https://reimagined-disco-g6q6ny1.pages.github.io/notebooks/mapping-exposure/mapping-exposure-population.html#create-the-maps):
```

#### Create the maps
The goal of this section is to create maps using the high resolution population layer aggregated at the subnational administrative boundary level 2. For this, we use the administrative boundary we used in [section 8.3.2](earthquake-overlay-admin2), when we studied earthquake intensity.  
```
At the beginning of [this notebook](https://github.com/worldbank/alternative-data-for-crisis/blob/port-activity/notebooks/physical-impact/damage-assessment.ipynb) there is a cross reference to [this section](https://reimagined-disco-g6q6ny1.pages.github.io/notebooks/mapping-and-monitoring-extent/mapping-and-monitoring-surface-change.html). 
Could you change it in a similar way I showed above? Let me know if it is not clear.

abrilr1604 committed:
add (Illustrative) to the title

abrilr1604 committed:
cross reference

abrilr1604 commented:
@msoltadeo  I made the change and added it to this PR! Hope it works 

I also added another update becauseI realized I forgot to add (Illustrative) to the title of one of the notebooks.. I was sure I did that two weeks ago 🤔 but it’s done now!

abrilr1604 committed:
update

msoltadeo reviewed (approved):
Looks good to me
